ente-io/bada-frame
# Cancel request

## Description
Adds logic at the caller level to cancel the request made , using the canceller provided.
Addes logic to photoswipe to call cancelRequest when user switches to a next  file for the prev file

## Test
-  check request is cancelled after the given time
- checked by switch to next video waiting for 5 sec for the first video to be canceled and then go back to see if a re-request is made
- switch between two video within 5 sec to check non on them are cancelled
- cancelled request dont trigger logError
